src/instance/instance.js

```diff
@@ -1,5 +1,4 @@
-// Import OpenLayers CSS and base Map and View definitions.
-import 'ol/ol.css';
+// Import OpenLayers base Map and View definitions.
```

Comment:
We'll be expecting the caller to arrange for the OpenLayers css to be available on the page.